Makefile

```diff
@@ -0,0 +1,22 @@
+
+all: klaw api
+
+klaw:
+	mvn clean install
+	mvn clean package
+
+api:
+	cd cluster-api && mvn clean install
+	cd cluster-api && mvn clean package
```

Comment:
Same as above
```suggestion
	cd cluster-api && mvn clean verify
```

Reply:
Question, shall we do `cd ..` after the command? I guess we want to return to the same directory we were before

Reply:
no need :) this will stay in the root directory and cd in only to run that command.